snippets/csharp/VS_Snippets_VBCSharp/VbRadconService/CS/MyNewService.cs

```diff
@@ -65,15 +65,15 @@ public MyNewService()
   //<Snippet3>
 	protected override void OnStart(string[] args)
 	{
-		eventLog1.WriteEntry("In OnStart");
+		eventLog1.WriteEntry("In OnStart.");
```

Comment:
This looks like it's formatted badly. Can we standardize on 4 spaces for indentation?

Reply:
Hi @BillWagner, thanks for your review - I've standardized the spacing in the MyNewService C# and VB files.